### PR TITLE
Tableau

### PR DESCRIPTION
**Pull request by ghislainfourny**

### CODE REVIEW

**Comment by wcandillon** [comment]
It's really good!!! There are some minor comments below and we need to embed the examples into the page.

You need to migrate the step by step instructions and move into the cellstore documentation maybe. If you are not keen on putting it into the cellstore documentation, move it at the end of the blog entry (after more insight into financial data.). Maybe a screen recording like Tableau does would be nice. 

The intro is great but the first paragraph should be removed maybe and the OData annoucement be made at the end of the intro. 
"Business users love their BI tools, so why would we force them to switch? " There is an implied proposition here that is not clear when you start to read but that is kinda clear after the intro. Also instead of saying "a format that Tableau supports", I would say "OData, a format that Tableau amongst other tools supports".

I would be cautious with using the term NoLAP. That's a lot of new words. If you do use it use the extended version too. Then I would reuse the same formulation from previous blog entries (hypercubes are the queries not the schema).

I would remove paragraph 3 and mention in the last one that we are focusing on making the cell store interoperable with OLAP clients.

I would remove the (Atom) part. Mention the existing output format available already and that we are proud to add odata atom to the mix.

Maybe link to the secxbrl and nttdata case studies at the end (no strong opinon).

**Comment by ghislainfourny** [comment]
@wcandillon back to you!

**Comment by wcandillon** [comment]
Very cool!

File: app/data/blog/tableau.md [hunk 1]
```diff
@@ -0,0 +1,92 @@
+<!--
+{
+    "title": "Integrating cells stores with Tableau",
+    "updated": "2015-06-23T18:00:00Z",
+    "tags": ["nolap", "tableau"],
+    "image": "/images/blog/tableau.png",
```
Comment (wcandillon) [inline app/data/blog/tableau.md]: The tableau.png file seems to be missing.

**Pull request merged**